app/views/timelines/_form.html.erb

```diff
@@ -42,6 +42,24 @@
       </div>
     </div>
   </div>
+  <div id="timeline_lti_share" class="row form-group container collapse <%= 'in' if Avalon::Authentication.lti_configured? && @timeline.visibility != 'private' %>">
```

Comment:
How about moving the if `lti_configured?` up a level to wrap this whole div?  Would the JS that shows/hides need to be changed?